app/src/main/java/com/codesoom/assignment/domain/User.java

```diff
@@ -0,0 +1,47 @@
+package com.codesoom.assignment.domain;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Entity
+@Getter
+@Builder
+@NoArgsConstructor
+@ToString
+@EqualsAndHashCode
+public class User {
+
+    @Id
+    @GeneratedValue
+    private Long id;
+
+    private String name;
+
+    private String email;
+
+    @JsonProperty(access = JsonProperty.Access.WRITE_ONLY)
+    private String password;
+
+    // @Builder를 위한 private 생성자
```

Comment:
클래스 시그니처 위에 `@Builder`를 선언하지 않고 생성자 바로 위에 `@Builder`를 선언하는 경우의 장점에 대해 고민해 보시고, 조사도 해 보세요.

```suggestion
    @Builder
```

Reply:
종립님의 글 참고하여 [정리](https://jdalma.github.io/docs/algorithmTheory/lombok/#builder-%ED%81%B4%EB%9E%98%EC%8A%A4%EC%97%90-%EC%9E%91%EC%84%B1%ED%95%98%EB%8A%94-%EA%B2%83%EA%B3%BC-%EB%A9%94%EC%84%9C%EB%93%9C%EC%97%90-%EC%9E%91%EC%84%B1%ED%95%98%EB%8A%94-%EA%B2%83%EC%9D%98-%EC%B0%A8%EC%9D%B4%EC%A0%90)해 봤습니다 ㅎㅎ